### ISSUE TITLE
Resize the side panels.

### ISSUE THREAD

**Issue by Gibbsdavidl** [maximilianh/cellBrowser#157]
Would be lovely to resize the two side panels, similar to how the heatmap can be resized.

**Comment by maximilianh** [comment]
Sorry for the delay: I agree that would be nice, but I am not sure I
have the time for it. Wouldn't a mouse over be good enough? The mouse
over could show the longer names. Don't I already have mouse overs in
place for most strings that can go over the panel?

BTW: do you have a website where I could look at an example where the
current dimensions are a problem?

**Comment by Gibbsdavidl** [comment]
Hi there,  Yeah, not a big deal... it's a wish list item.  Sorry our data is private right now.. but here's a screen shot.  You can see some of the cell type labels are so long they wrap around.. maybe OCD of me, but I really want to stretch out the panel so they don't wrap.

Maybe I can figure it out!  ;-)

![image](https://user-images.githubusercontent.com/561865/73967539-eb834b80-48cc-11ea-90a8-5a8888d5cae0.png)

**Comment by maximilianh** [comment]
If you could suggest a good javascript library for doing stretching, that
would be a big help. I spent a few hours one day, just to go in circles.
Ideally something small with minimal dependencies. I ended up writing it
myself for the heatmap but it wasn't very general.

Possibly you could just copy what I did for the heatmap ...? My
implementation is not great at all though, you may want to restructure it a
little into two functions...

>

**Comment by Gibbsdavidl** [comment]
I'm getting advice from the web dev in our lab.

**Comment by maximilianh** [comment]
Ah, good. What is the advice?

**Comment by Gibbsdavidl** [comment]
Sorry, I don't have any real solutions yet.  *But*  a quick solution would be to have a configuration parameter (in the conf file) that sets the width of the side panels..   I think they're hard coded right now.

**Comment by maximilianh** [comment]
Good idea. In the develop branch, you can now set in cellbrowser.conf:
metaBarWidth = 400

I also increased the left sidebar to 270 pixels from 200, inspired by a
discussion with @mspeir. 250 pixels is the size of Gmail's left sidebar.

**Comment by maximilianh** [comment]
Can you use the git develop branch? Otherwise, I am happy to do a
pre-release in pip.

**Comment by Gibbsdavidl** [comment]
Thanks! I'll give it a try.

**Comment by Gibbsdavidl** [comment]
Seems to work well... but only the left-side-panel is changing.  Maybe would be nice to have the right side move too.  Could make larger violin plots, which would be nice.  Thanks though! This is good.

**Comment by maximilianh** [comment]
Hi Debashish, right now that's not possible. Can yo tell me why you want to
change the size of the right panel? Is there are dataset where the size of
it is a problem?

**Comment by devashishmohanta** [comment]
Hi @maximilianh,
Yes, you get it right sometimes the label annotation is larger than the LegendBar width size which makes it difficult to read as the text gets wrapped around the color boxes as mentioned by @Gibbsdavidl above. If it can be fixed somehow it will be really helpful.
Thanks

**Comment by matthewspeir** [comment]
I recently implemented a fix that should help with this. While the labels still wrap, they at least don't wrap under the colors:

<img width="197" height="316" alt="Image" src="https://github.com/user-attachments/assets/4b124ab6-5564-40e0-823b-2c30cdc9147c" />

It's currently only on the develop branch, but hopefully it'll make it out into a full release soon. 

Also, @devashishmohanta and @Gibbsdavidl are you all still using the Cell Browser? If so, feel free to reach out with any feedback or requests or anything.
